src/index.html

```diff
@@ -17,11 +17,388 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The HTML file is linking to an SCSS file (`styles/style.scss`). Browsers cannot interpret SCSS directly, so you need to compile this SCSS file into a CSS file and update the link to point to the compiled CSS file.